program/rust/src/tests/test_upd_price.rs

```diff
@@ -41,13 +41,13 @@ fn test_upd_price() {
     let mut funding_setup = AccountSetup::new_funding();
     let funding_account = funding_setup.as_account_info();
 
-    let mut price_setup = AccountSetup::new::<PriceAccount>(&program_id);
```

Comment:
i suggest copy/pasting this test code to also test the vanilla `PriceAccount` codepath. Both of them should work for the moment (and we can delete the copy/paste when we delete the old PriceAccount)